### PR TITLE
Feature/delete unused manipulations

### PR DESCRIPTION
**Pull request by pstaeck**
In this PR we have removed the unused manipulations.

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* README update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: .github/workflows/build-python.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
   push: # Apply to all branches
 
 env:
-  PYTHON_VERSION: 3.8.10
```
Comment (jannast) [inline .github/workflows/build-python.yml:8]: I would expect the python version to be 3.8.10. Why did you change this? It was not requested as I know.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,16 @@ All notable changes to the t2iapi module will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
 
+### Update
+
+- update used PYTHON_VERSION: 3.8
+
+### Removed
```
Comment (jannast) [inline CHANGELOG.md:13]: If you remove / change manipulations then you have to update the BASE_PACKAGE_VERSION too.
BASE_PACKAGE_VERSION should be 2.0.0

File: .github/workflows/build-python.yml (hunk 1 above)
Comment (pstaeck) [reply .github/workflows/build-python.yml:8]: If I understand it correctly there was a migration to the latest ubuntu 22.04 and the "build-python" step failed with:

The version '3.8.10' with architecture 'x64' was not found for Ubuntu 22.04.
The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json

File: CHANGELOG.md (hunk 2 above)
Comment (pstaeck) [reply CHANGELOG.md:14]: done

Comment (pstaeck) [reply CHANGELOG.md:14]: as discussed it is not necessary to list the version updates here

File: .github/workflows/build-python.yml (hunk 1 above)
Comment (pstaeck) [reply .github/workflows/build-python.yml:8]: as discussed, setting the python version to 3.8 is not critical and has no direct effect on our RBT release

**Review by jannast — APPROVED** [review]
REVIEW_OK
